### ISSUE TITLE
White battery icon (alt status bar) and inverted screenshot in Night mode 

### ISSUE THREAD

**Issue by hritsh** [koreader/koreader#7750]
* KOReader version: 2021.05
* Device: Kindle PW2

Not really sure if this counts as a bug, but my ocd was eating me

<img src="https://user-images.githubusercontent.com/65954042/119309496-708ba300-bc7f-11eb-99ac-fd9001dd5cf7.jpeg" alt="drawing" width="500"/>

I took a screenshot through koreader too, but it was inverted for some reason

<img src="https://user-images.githubusercontent.com/65954042/119309836-eabc2780-bc7f-11eb-9575-3eed251b0b5c.png" alt="drawing" width="500"/>

**Comment by NiLuJe** [comment]
Yep, somewhat out of our hands. For images we control, we actually invert 'em ourselves to counteract nightmode, but this one *not* one we control ;).

**Comment by hritsh** [comment]
> Yep, somewhat out of our hands. For images we control, we actually invert 'em ourselves to counteract nightmode, but this one is *not* one we control ;).

ohh so is it like an external api or something? (sorry im kinda new to open source 😅)

**Comment by NiLuJe** [comment]
No, the engine does, and that part of the engine has no idea whether nightmode is on or off.

**Comment by hritsh** [comment]
> No, the engine does, and that part of the engine has no idea whether nightmode is on or off.

oh i get it now.. thanks for clarifying :)

**Comment by virxkane** [comment]
@NiLuJe 
> No, the engine does, and that part of the engine has no idea whether nightmode is on or off.

engine draw icons that you specified in `LVDocView::setBatteryIcons()` https://github.com/koreader/crengine/blob/b20e7717ef1db36b9bddb00e0b893f6082da7d29/crengine/src/lvdocview.cpp#L1687
So, when you switch to night mode, you can change this icons to night-mode version and so on.
